doc.ddoc

```diff
@@ -1,3 +1,4 @@
+ROOT_DIR =
```

Comment:
> and I suggested to define it as $(ROOT)/.

No, that would make it pointless. It is only useful because instead of `./`, it will be an empty string.